src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/Microsoft.NET.Build.Tasks.UnitTests.csproj

```diff
@@ -20,9 +20,13 @@
     <PackageReference Include="Microsoft.Build.Utilities.Core" />
     <PackageReference Include="Microsoft.Extensions.DependencyModel" />
     <PackageReference Include="NuGet.ProjectModel" />
+    <PackageReference Include="System.Security.Cryptography.X509Certificates" />
```

Comment:
You shouldn't need a package reference to X509Certificates. The package hasn't shipped since 2017 and will soon be marked as deprecated. You don't need the PackageReference on .NET Framework either as the X509Certificates types are inbox.